.github/actions/check_for_stc/action.yml

```diff
@@ -13,6 +13,7 @@ runs:
         from os.path import exists, join
         pyccel_loc = pyccel.__path__[0]
         print(pyccel_loc)
+        print("==========================================------------------> ${{ inputs.editable_string }}")
```

Comment:
You don't have access to this here. It is defined in one (but not all) of the workflows that calls this action. It is not an input to this action